changelog.txt

```diff
@@ -1,3 +1,37 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Option to (dis-)allow accelerated video drivers (#8819)
+Feature: Display refresh rate game option (#8813)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
+Add: Make NewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
+Add: Malaysia Ringgit as Currency (#8783)
+Add: Filter for "engines only" in build train window (#8733)
+Change: De-limit framerate window's framerate (#8772)
+Change: Clarify what effect town interactions have (#8744)
```

Comment:
Minor enough to remove?

Reply:
Nah, it is fine. We shouldn't make this too difficult for ourselves.